Automattic/wp-calypso
# Don't show intro pricing text on multiyear dropdown for premium domains

Related to #
This is a fix to the conversation from p1712775872203739-slack-C0BNMNMNG
A follow-up to this patch D141868-code

## Proposed Changes
Remove intro pricing text on multiyear dropdowns on the checkout page for domains

**Before**
<img width="854" alt="Screenshot 2024-04-12 at 09 28 57" src="https://github.com/Automattic/wp-calypso/assets/11347472/1c388464-f691-46cc-ad32-35593dd225f3">


**After**
<img width="854" alt="Screenshot 2024-04-12 at 09 32 01" src="https://github.com/Automattic/wp-calypso/assets/11347472/cc2c5724-1f5c-43a6-8ede-283e0f95144c">



## Testing Instructions
- Apply patch here D141868-code (Follow instructions there for sandboxing)
- Use the live link in this PR and try to purchase a premium domain like firefighter.global
- Verify that you see a drop-down for multi-year domain purchases.
- Verify that the dropdown doesn't have additional pricing text below each number of years that can be registered e.g 
US$xx first year then US$xx per year



*

## Pre-merge Checklist
